Doc/library/base64.rst

```diff
@@ -53,21 +53,23 @@ The modern interface provides:
    Encode the :term:`bytes-like object` *s* using Base64 and return the encoded
    :class:`bytes`.
 
-   Optional *altchars* must be a :term:`bytes-like object` of at least
-   length 2 (additional characters are ignored) which specifies an alternative
-   alphabet for the ``+`` and ``/`` characters.  This allows an application to e.g.
-   generate URL or filesystem safe Base64 strings.  The default is ``None``, for
-   which the standard Base64 alphabet is used.
+   Optional *altchars* must be a :term:`bytes-like object` of length 2 which
+   specifies an alternative alphabet for the ``+`` and ``/`` characters.
+   This allows an application to e.g. generate URL or filesystem safe Base64
+   strings.  The default is ``None``, for which the standard Base64 alphabet is used.
+
+   Raises an :exc:`AssertionError` if the length of *altchars* is not 2.  Raises a
```

Comment:
It's unfortunate that the function uses an `assert` statement here, because it may not always raise.

Can we change this to ``May assert or raise a :exc:`ValueError` if the length ...``? If you disable asserts then you get ValueError from the next statement.